source/_integrations/squeezebox.markdown

```diff
@@ -8,9 +8,11 @@ ha_iot_class: Local Polling
 ha_domain: squeezebox
 ---
 
-The `squeezebox` platform allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
+The `squeezebox` platform allows you to control a [Logitech Squeezebox](https://en.wikipedia.org/wiki/Squeezebox_%28network_music_player%29) music player from Home Assistant. This lets you control Squeezebox hardware like the Classic, Transporter, Duet, Boom, Radio and Touch and of software players like [Squeezelite](https://github.com/ralph-irving/squeezelite), [SoftSqueeze](http://softsqueeze.sourceforge.net/), [SqueezePlayer](https://play.google.com/store/apps/details?id=de.bluegaspode.squeezeplayer) and [SqueezeSlave](https://forums.slimdevices.com/showthread.php?93607-ANNOUNCE-Squeezeslave-1-2-released).
 
-To add your Squeezebox player to your installation, add the following to your `configuration.yaml` file:
+You can add the Squeezebox integration through the frontend configuration menu. This is the preferred method.
```

Comment:
Please describe how to set it up via the frontend. What does a user, who has never worked with Home Assistant, have to do?

Reply:
Thanks, this is a helpful way to think about it. I just pushed an updated description. Let me know what you think.

Reply:
Awesome!